libs/design/src/scss/README.md

```diff
@@ -1,2 +0,0 @@
-`daff-util.scss`
-Utility library that contains all the different mixins and functions.
```

Comment:
needs docs

Reply:
Resolved in 6e0d8e5f09e11ac7c0fd9050ac33eb6a213be97d